bundle/store.go

```diff
@@ -484,9 +485,14 @@ func writeModules(ctx context.Context, store storage.Store, txn storage.Transact
 		}
 	}
 
+	if schema != nil {
+		compiler = compiler.WithSchema(schema)
```

Comment:
Per comment above, we can do this outside.

Reply:
Done